<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zjhRegisterWindow</class>
 <widget class="QWidget" name="zjhRegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>312</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>注册帐号</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QLabel" name="registerLabel1">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>     帐号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="registerLineText1">
       <property name="font">
        <font>
         <family>MS UI Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QLabel" name="registerLabel2">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>     密码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="registerLineText2">
       <property name="font">
        <font>
         <family>MS UI Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QLabel" name="registerLabel3">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>确认密码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="registerLineText3">
       <property name="font">
        <font>
         <family>MS UI Gothic</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="okRegisterBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cencelRegisterBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
